frontend/src/components/mine/Tailings/MineTailingsInfo.js

```diff
@@ -200,34 +240,41 @@ export class MineTailingsInfo extends Component {
                           doc
                         )
                       }
-                  >
-                    <img style={{ padding: "5px" }} src={GREEN_PENCIL} />
-                  </Button>
-                  <Popconfirm placement="topLeft" title={`Are you sure you want to delete ${  doc.exp_document_name  }?`} onConfirm={(event) => this.removeReport(event, doc.exp_document_guid)} okText="Delete" cancelText="Cancel">
-                    <Button ghost type='primary'>
-                      <Icon type="minus-circle" theme="outlined" />
+                    >
+                      <img style={{ padding: "5px" }} src={GREEN_PENCIL} />
                     </Button>
-                  </Popconfirm>
-                </Col>
-              </Row>
-              <hr />
-            </div>
+                    <Popconfirm
+                      placement="topLeft"
+                      title={`Are you sure you want to delete ${doc.exp_document_name}?`}
+                      onConfirm={(event) => this.removeReport(event, doc.exp_document_guid)}
+                      okText="Delete"
+                      cancelText="Cancel"
+                    >
+                      <Button ghost type="primary">
+                        <Icon type="minus-circle" theme="outlined" />
+                      </Button>
+                    </Popconfirm>
+                  </Col>
+                </Row>
+                <hr />
+              </div>
             ))}
-          <div key='0'>
+          <div key="0">
             <Row gutter={16} justify="center" align="top">
               <Col span={8} align="left">
                 <Button
                   className="full-mobile"
                   type="secondary"
-                  onClick={(event) => 
+                  onClick={(event) =>
                     this.openAddReportModal(
                       event,
                       this.handleAddReportSubmit,
                       ModalContent.ADD_TAILINGS_REPORT,
                       this.props.mineTSFRequiredReports
-                    )}
+                    )
+                  }
                 >
-+ Add TSF Report
+                  + Add TSF Report
```

Comment:
Use the title as the button content, I believe its `ModalContent.ADD_TAILINGS_REPORT`, that way if the name changes it can be updated in one spot.

Reply:
Changed.